docs/source/develop.rst

```diff
@@ -86,23 +86,19 @@ Contributions to Dask can then be made by submitting pull requests on GitHub.
 Install
 ~~~~~~~
 
-You may want to install larger dependencies like NumPy and Pandas using a
-binary package manager like conda_.  You can skip this step if you already
-have these libraries, don't care to use them, or have sufficient build
-environment on your computer to compile them when installing with ``pip``::
+To build the library you can install the necessary requirements using
+``pip`` or ``conda``::
```

Comment:
Perhaps make `conda` a link to `https://conda.io`?

Reply:
Stupid question you know to hyperlink code format e.g. ``conda<https://conda.io>``_ doesn't work

Reply:
I don't think rst supports this. I wouldn't put it in code format, I'd leave it as a normal link.